ansible_collections/arista/avd/plugins/filter/convert_schema.py

```diff
@@ -8,37 +8,37 @@
 from ansible_collections.arista.avd.plugins.plugin_utils.schema.avdtojsonschemaconverter import AvdToJsonSchemaConverter
 
 
-def convert_schema(schema: dict, type: str):
+def convert_schema(schema_id: str, type: str):
     """
     The `arista.avd.convert_schema` filter will convert AVD Schema to a chosen output format.
 
     Parameters
     ----------
-    schema : dict
-        Input AVD Schema
+    schema_id : str, ["eos_cli_config_gen" , "eos_designs"]
+        ID of AVD Schema
     type : str, ["documentation", "jsonschema"]
         Type of schema to convert to
 
     Returns
     -------
-    dict
+    dict | list
```

Comment:
I am wondering if this is very good to have a return type being either a list or a dict now.. should we maybe separate in two separate functions for more clarity?

Reply:
I suspect there will be many changes to the documentation area as we get through the eos_designs schemas, so I will split it next time we have to rework it. Adding TODO for now.